XYZDrivers/src/main/java/com/xyzdrivers/services/InsertClaimService.java

```diff
@@ -0,0 +1,48 @@
+/**
+ * @file InsertClaimSService.java
+ * @author Nathan
+ * @created 02/11/17
+ * @modified 03/11/17
+ * @notes -
+ */
+
+package com.xyzdrivers.services;
+
+import com.xyzdrivers.models.Claim;
+import java.sql.Connection;
+import java.sql.Date;
+import java.sql.PreparedStatement;
+import java.sql.SQLException;
+
+public class InsertClaimService {
+
+    private Connection con;
+    
+    public InsertClaimService(Connection con) {
+        this.con = con;
+    }
+
+    public void InsertClaim(Claim c) {

```

Comment:
Sorry, should've picked up before.
Needs validation checks, eg
- Check that claim isn't null 
- Check each property of claim for null/invalid data
And throw IllegalArgumentException for each.